config.lua

```diff
@@ -0,0 +1,302 @@
+Config.MinimumPrice = 1000
+Config.GardenAndPoolTaxes = true
+Config.Taxes = { -- % of the property price (if price is 1000$ and tax is 10%, then the player pays 1100$), applies to both rent (repeatedly) and buy (once)
+    Garden = 3,
+    Pool = 5,
+    General = 10,
+}
+
+Config.Properties = {
+    Marker = {
+        type = 25,
+        offsetZ = -0.95,
+        color = {r = 82, g = 145, b = 170, a = 155},
+        scale = {x = 0.8, y = 0.8, z = 0.8},
+    },
+
+    Blip = {
+        Owned = {
+            sprite = 40,
+            color = 38,
+            scale = 0.7,
+        },
+        Rent = {
+            sprite = 40,
+            color = 25,
+            scale = 0.7,
+        },
+        Garage = {
+            sprite = 357,
+            color = 3,
+            scale = 0.7,
+        },
+    },
+}
+
+Config.Apartments = {
+    Marker = {
+        type = 25,
+        offsetZ = -0.95,
+        color = {r = 82, g = 145, b = 170, a = 155},
+        scale = {x = 0.8, y = 0.8, z = 0.8},
+    },
+    Blip = {
+        sprite = 475,
+        color = 3,
+        scale = 0.7,
+    },
+    weight = 200000,
+    slots = 40,
+}
+
+Config.defaultapartment = 1
+Config.apartmentlocations = {
+    {
+        name = "Alta Street Apartments",
+        coords = vector4(-271.1, -957.5, 31.22, 291.66),
+        IPL = "LowEnd", -- Config.IPLS["LowEnd"]
+    },{
+        name = "Morningwood Blvd",
+        coords = vector4(-1288.52, -430.51, 35.15, 124.81),
+        IPL = "LowEnd",
+    },{
+        name = "Tinsel Towers",
+        coords = vector4(-619.29, 37.69, 43.59, 181.03),
+        IPL = "LowEnd",
+    },{
+        name = "Fantastic Plaza",
+        coords = vector4(291.517, -1078.674, 29.405, 270.75),
+        IPL = "LowEnd",
+    },
+}
+
+Config.IPLS = { -- 'Ipls' can just be interiors that aren't proper IPLs, but are still interiors
+    ["AltaStreet"] = {
+        ipl = false,
+        coords = {
+            entrance = vector4(-271.87, -940.34, 92.51, 70),
+            wardrobe = vector4(-277.79, -960.54, 86.31, 70),
+            stash = vector4(-272.98, -950.01, 92.52, 70),
+            logout = vector3(-283.27, -959.68, 70),
+        }
+    },
+    ["apa_v_mp_h_01_a"] = {
+        ipl = "apa_v_mp_h_01_a",
+        coords = {
+            entrance = vector4(-786.866, 315.764, 217.638, 160),
+            wardrobe = vector4(-786.866, 315.764, 217.638, 160),
+            stash = vector4(-786.866, 315.764, 217.638, 160),
+            logout = vector3(-786.866, 315.764, 217.638),
+        }
+    },
+    ["LowEnd"] = {
+        ipl = false,
+        coords = {
+            entrance = vector4(265.95, -1007.41, -101.01, 2.71),
+            wardrobe = vector4(259.76, -1003.63, -99.01, 182.24),
+            stash = vector4(265.8, -999.47, -99.01, 268.16),
+            logout = vector3(262.9, -1003.09, -99.01),
+        }
+    }
+}
+
+Config.Shells = {
+    ["empty1"] = {
+        prop = "", -- shell prop i guess idk

```

Comment:
Unclear what prop is?

Reply:
now that i think of it, it could just be "shell"

Reply:
modified to shell for clarity